tests/braille-specs/en-ueb.yaml

```diff
@@ -94,6 +94,12 @@ tests:
   - ',-,,grt'
 - - OUT OF TOWN
   - ',,,| ( t{n,'''
+- - '[open TN]every'
+  - '`.<e'
+  - {xfail: 'transcriber note not defined'}
```

Comment:
@BueVest There seems to be a `transnote` defined in the table:

```
begemph transnote 4-46-126
endemph transnote 4-46-354
```

And there are also `trans1`, `trans2`, `trans3`, `trans4`, `trans5`